src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -1,3 +1,45 @@
+"""
+    NormInfinityBridge{T}
+
+The `NormInfinityCone` is representable with LP constraints, since ``t \\ge \\max_i |x_i|`` if and only if ``t \\ge x_i`` and ``t \\ge -x_i`` for all ``i``.
+"""
+struct NormInfinityBridge{T} <: AbstractBridge
+    ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
+end
+function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorOfVariables, s::MOI.NormInfinityCone) where T
+    return bridge_constraint(NormInfinityBridge{T}, model, MOI.VectorAffineFunction{T}(f), s)
+end
+function bridge_constraint(::Type{NormInfinityBridge{T}}, model, f::MOI.VectorAffineFunction{T}, s::MOI.NormInfinityCone) where T
+    f_scalars = MOIU.eachscalar(f)
+    t = f_scalars[1]
+    x = f_scalars[2:end]
+    d = MOI.dimension(s) - 1
+    ge_index = MOI.add_constraint(model, MOIU.operate(vcat, T, t - x, t + x), MOI.Nonnegatives(2d)) # TODO fix
```

Comment:
@blegat how should I do the `t - x` here?

Reply:
This is different from my previous question that you answered, because this is a vector.